src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/JsonConverterEnum.cs

```diff
@@ -21,14 +21,14 @@ public override bool CanConvert(Type type)
         [PreserveDependency(
             ".ctor(System.Text.Json.Serialization.Converters.EnumConverterOptions)",
             "System.Text.Json.Serialization.Converters.JsonConverterEnum`1")]
-        public override JsonConverter CreateConverter(Type type, JsonSerializerOptions options)
+        public override JsonConverter CreateConverter(Type type, JsonSerializerOptions? options)
```

Comment:
I am still not sure why `JsonSerializerOptions` should be nullable here/other converters.
cc @steveharter, @layomia

Reply:
The Converter APIs should not have `options` be nullable.

This includes The Read/Write methods on `JsonConverter<T>`. Is there anything causing it to be null?

Reply:
It is same as https://github.com/dotnet/runtime/pull/528#discussion_r355806764, made them non null